apisix/core/ctx.lua

```diff
@@ -96,6 +96,9 @@ do
             elseif key == "service_id" then
                 val = ngx.ctx.api_ctx and ngx.ctx.api_ctx.service_id
 
+            elseif key == "consumer_name" then
+                val = ngx.ctx.api_ctx and ngx.ctx.api_ctx.consumer_id
```

Comment:
It always confuses me, `name` or `id`? we should choose a good name

Reply:
`consumer_id` and `consumer_name` are the same string type value. After processing the "consumer_id" in the core code "ctx.lua" file here, all users will see the "consumer_name".